### PR TITLE
Add accessibility importer

### PR DESCRIPTION
**Pull request by frwickst**
Note that this branch is based on top of the service importer! The same commits exists in both branches.

The new commits for this PR starts from `Add ptv_id to Unit model`.

### CODE REVIEW

File: services/management/commands/turku_service_import/accessibility.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                num_of_updated_units += 1
+
+        print('Updated {} units.'.format(num_of_updated_units))
+
+    def _import_unit_accessibility_properties(self):
+        property_syncher = ModelSyncher(UnitAccessibilityProperty.objects.all(), lambda obj: obj.id)
+        num_of_imports = 0
+
+        # For caching unit ids that are not present in the database
+        unit_skip_list = []
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/accessibility.py]: A `set` would be better for this, of course it doesn't matter in practice here.

File: services/management/commands/turku_service_import/accessibility.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        variables = get_ar_resource('accessibility/variables')
+        for variable in variables:
+            variable_name = variable.get('variableName')
+            variable_id = variable.get('variableId')
+            if not (variable_name and variable_id):
+                continue
+            accessibility_variable, created = AccessibilityVariable.objects.get_or_create(
+                id=variable_id,
+                name=variable_name,
+            )
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/accessibility.py]: I don't know much about the data in question, but might be still better to do
```
AccessibilityVariable.objects.update_or_create(
    id=variable_id,
    defaults={'name': variable_name}
)
```

so it won't break if a name happens to change?

File: services/management/commands/turku_service_import/accessibility.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                print("Unit {} does not exist, skipping".format(unit_id))
+                unit_skip_list.append(unit_id)
+                continue
+
+            # Make sure that the variable exists
+            if accessibility_variable_id not in self._accessibility_variables:
+                print("No variable {}, skipping".format(unit_id))
+                continue
+
+            # Create of update the property including its associated value
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/accessibility.py]: typo

File: services/management/commands/turku_service_import/accessibility.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
+from munigeo.importer.sync import ModelSyncher
+
+from services.management.commands.turku_service_import.utils import get_ar_resource, get_ar_servicepoint_resource, \
+    get_ar_servicepoint_accessibility_resource
+from services.models import Unit, AccessibilityVariable, UnitAccessibilityProperty
+
+VERBOSITY = False
+LOGGER = None
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/accessibility.py]: These don't seem to be used anywhere.

File: services/management/commands/turku_service_import/accessibility.py [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+
+class AccessibilityImporter:
+    def __init__(self, verbosity=True, logger=None):
+        self.verbosity = verbosity
+        self.logger = logger
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/accessibility.py]: We should probably get rid of that verbosity thing as it's not used anywhere. And maybe start using the logger? :)

File: services/management/commands/turku_service_import/accessibility.py (hunk 2 above)
Comment (frwickst) [reply services/management/commands/turku_service_import/accessibility.py]: You a correct, this should have been made with `defaults`

File: services/management/commands/turku_service_import/accessibility.py (hunk 4 above)
Comment (frwickst) [reply services/management/commands/turku_service_import/accessibility.py]: Damnit, relics from the Helsinki code again, thought I had managed to get those out this time 😄

File: services/management/commands/turku_service_import/accessibility.py (hunk 5 above)
Comment (frwickst) [reply services/management/commands/turku_service_import/accessibility.py]: Yeah, lets do that

**Pull request merged**